src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx

```diff
@@ -8,37 +8,48 @@ import {faRedoAlt} from '@fortawesome/free-solid-svg-icons'
  */
 class SecondaryTableAttribute extends TableAttribute {
   nullable: boolean;
-  defaultValue: string;
+  defaultValue?: string;
 
   constructor(attributeName: string,
     attributeType: TableAttributeType,
     nullable: boolean,
-    defaultValue: string,
+    defaultValue?: string,
     stringTypeAttributeLengthInfo?: number,
     enumOptions?: Array<string>,
     decimalNumDigits?: number,
     decimalNumDecimalDigits?: number
     ) {
     super(attributeName, attributeType, stringTypeAttributeLengthInfo, enumOptions, decimalNumDigits, decimalNumDecimalDigits);
     this.nullable = nullable;
-    this.defaultValue = defaultValue;
+    this.defaultValue = defaultValue === null || defaultValue === 'null' ? undefined : defaultValue;
   }
 
   static getAttributeLabelBlock(secondaryTableAttribute: SecondaryTableAttribute, resetToNullCallback: any) {
     const typeString = super.getTypeString(secondaryTableAttribute);
+    var resetButtonText = 'nullable';
+
+    // I don't think this detects is correctly
```

Comment:
```suggestion
    // I don't think this detects it correctly
```

Reply:
I fix this one, at least with the test cases I tried